src/main/java/AacademyCourse/demohomeworkmodule14/controller/UserController.java

```diff
@@ -0,0 +1,43 @@
+package AacademyCourse.demohomeworkmodule14.controller;
+
+import AacademyCourse.demohomeworkmodule14.entity.User;
+import AacademyCourse.demohomeworkmodule14.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    @Autowired
+    UserService userService;
+
+    @GetMapping("/sayHello")
```

Comment:
this method wasn't mentioned in the homework

Reply:
conversation resolved